client/verta/verta/client.py

```diff
@@ -4066,6 +4066,60 @@ def get_deployed_model(self):
         status = self.get_deployment_status()
         return deployment.DeployedModel.from_url(status['url'], status['token'])
 
+    def download_deployment_crd(self, download_to_path, path=None, token=None, no_token=False):
```

Comment:
Crd => yaml. Let's not have the user worry about details.

Reply:
@conradoverta I'm not sure what "crd => yaml" is conveying.
Is the suggestion to rename the method to `download_deployment_yaml`? Or automatically append ".yaml" to the destination filename? Or to return a dict object from reading the YAML?

Reply:
Rename